tools/generate_static_provisioning_yamls.sh

```diff
@@ -84,7 +84,7 @@ spec:
     - ${accessmode}
   claimRef:
     name: pvc-${volname}
-    namespace: ibm-spectrum-scale-csi
+    namespace: <PVC namespace>
```

Comment:
Lets not use the claimRef approach for binding pv to pvc. Remove the reference for pv instead use volumeName: <replace with pv name> in pvc

Reply:
Done